futures-channel/tests/mpsc.rs

```diff
@@ -34,29 +34,29 @@ fn send_recv_no_buffer() {
 
     // Run on a task context
     let f = lazy(move || {
-        assert!(tx.flush().unwrap().is_ready());
-        assert!(tx.poll_ready().unwrap().is_ready());
+        assert!(tx.flush(&mut TaskContext).unwrap().is_ready());
```

Comment:
`TaskContext` isn't the name of a variable in this scope-- is this just a WIP artifact?

Reply:
This is using the prelude (I don't really like preludes, but I figured I'd use it... maybe it would confuse someone and convince them preludes were bad? j/k).